Solution1/BusinessLayer/Service/WeatherService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using AutoMapper;
+using BusinessLayer.Abstract;
+using BusinessLayer.DTOs;
+using DataAccessLayer.Abstract;
+using DataAccessLayer.Model;
+using DataAccessLayer.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Service
+{
+    public class WeatherService : IWeatherServise
+    {
+        readonly IWetherRepository _wetherRepository;
+        readonly IMapper _mapper;
+
+        public WeatherService(IWetherRepository wetherRepository, IMapper mapper) 
+        { 
+            _wetherRepository = wetherRepository; 
+            _mapper = mapper;
+        }
+
+        public string GetByCityName(string CityName)
```

Comment:
in C# lower camel case used for names of variables

Reply:
+